src/quo2/components/switchers/utils.cljs

```diff
@@ -0,0 +1,33 @@
+(ns quo2.components.switchers.utils
+  (:require [utils.i18n :as i18n]))
+
+(defn subtitle
+  [type {:keys [photos]}]
+  (case type
```

Comment:
Same as another comment I made above, if we don't add a default value, `case` will throw an exception and in production builds this means the app will crash. It's better to have a default value. Better have a `nil` or empty string being returned than a total crash on the user.